lib/_http_client.js

```diff
@@ -129,6 +129,12 @@ function ClientRequest(options, cb) {
                       self._renderHeaders());
   }
 
+  this._ended = false;
+  this.res = null;
+  this.aborted = 0;
+  this.timeoutCb = null;
+  this.upgradeOrConnect = false;
```

Comment:
@mcollina I'm sorry but I'm not sure I get it right, do you mean [this comment](https://github.com/nodejs/node/blob/master/lib/_http_client.js#L394-L396)?